### PR TITLE
Rollout improved example converter

### PR DESCRIPTION
**Pull request by t0yv0**
Rolls out improved TF example converter.

### CODE REVIEW

File: sdk/dotnet/IamUserPolicyAttachment.cs [hunk 1]
```diff
@@ -11,56 +11,6 @@ namespace Pulumi.Minio
 {
     /// <summary>
     /// ## Example Usage
-    /// 
-    /// &lt;!--Start PulumiCodeChooser --&gt;
```
Comment (iwahbe) [inline sdk/dotnet/IamUserPolicyAttachment.cs:15]: Looks like a regression here.

File: sdk/java/src/main/java/com/pulumi/minio/IamUserPolicyAttachment.java [hunk 2]
```diff
@@ -16,68 +16,6 @@
...
- * import com.pulumi.minio.IamUserPolicyAttachment;
- * import com.pulumi.minio.IamUserPolicyAttachmentArgs;
- * import java.util.List;
- * import java.util.ArrayList;
- * import java.util.Map;
- * import java.io.File;
- * import java.nio.file.Files;
- * import java.nio.file.Paths;
- * 
- * public class App {
```
Comment (iwahbe) [inline sdk/java/src/main/java/com/pulumi/minio/IamUserPolicyAttachment.java:38]: Likewise here.

**Review by iwahbe — APPROVED** [review]

**Comment by t0yv0** [comment]
```
jq '.|select(.ConversionResults.typescript.FailureSeverity==2)|.ConversionResults.typescript.FailureInfo' cov/byExample.json  | sort | uniq -c                                                                                          ~/bugs/ci-mgmt-889/pulumi-minio
   1 "Duplicate resource \"minio_iam_group_policy_attachment\" configuration: A minio_iam_group_policy_attachment resource named \"developer\" was already declared at /e1.tf:23,1-57. Resource names must be unique per type in each module."
   1 "Duplicate resource \"minio_iam_user_policy_attachment\" configuration: A minio_iam_user_policy_attachment resource named \"developer\" was already declared at /e0.tf:23,1-56. Resource names must be unique per type in each module."
```

**Comment by t0yv0** [comment]
Due to https://github.com/pulumi/pulumi-converter-terraform/issues/146

**Pull request merged**